### ISSUE TITLE
Load model using BertClassificationPredictor but cannot change max_seq_len

### ISSUE THREAD

**Issue by flysavage** [kaushaltrivedi/fast-bert#100]
In the method `get_learner()` from `BertClassificationPredictor`, the `max_seq_length` is set to 512 even I used a different `max_seq_length` while I was training. And this would make the model different before and after saving.

**Comment by kaushaltrivedi** [comment]
You are right. I will update this to accept max_seq_length as a parameter.

**Comment by ryanzhang-cbi** [comment]
I thought you did this for a purpose. thanks for your reply!

**Comment by flysavage** [comment]
I thought you did this for a purpose. thanks for your reply!
